playbooks/slave_open_ports.yml

```diff
@@ -0,0 +1,11 @@
+  - name: Secure Slave
```

Comment:
sure I thought it was good to leave that separately, one is for the basic host setup, the other was for firewalling. But fine for me to merge them.